src/sentry/static/sentry/app/views/performance/vitalsCards.tsx

```diff
@@ -37,12 +38,35 @@ type Props = {
   showVitalPercentNames?: boolean;
   showDurationDetail?: boolean;
   hasCondensedVitals?: boolean;
+  projects: Project[];
 };
 
-export default function VitalsCards(props: Props) {
-  const {eventView, organization, location} = props;
+// Temporary list of platforms to only show web vitals for.
+const VITALS_PLATFORMS = [
+  'javascript',
+  'javascript-react',
+  'javascript-angular',
+  'javascript-angularjs',
+  'javascript-backbone',
+  'javascript-ember',
+  'javascript-gatsby',
+  'javascript-vue',
+];
```

Comment:
You could add an export, and then import `frontend` from the platformCategories module.

Reply:
from above `which is why I've brought in the list from platformCategories instead of exporting it`, heh